templates/database.yml.epp

```diff
@@ -33,3 +34,6 @@
   password: <%= stdlib::to_ruby($password) %>
 <% } -%>
   pool: <%= $db_pool %>
+<% $extra_options.each |String $k, String $v| { -%>
+  <%= $k %>: <%= $v %>
```

Comment:
Should this also use `stdlib::to_ruby` to ensure things are quoted if needed? I don't trust unquoted strings with random input.